### DISCUSSION TITLE
Some info

### DISCUSSION THREAD

**Discussion by velea18** [NoahZinsmeister/web3-react#433] [Q&A]
Hello. Does someone has a more detailed example for this version of web3-react? I m still learning react and next and some things i don't really know how to achieve. Like, after you connect with a provider, metamask let s say, you have a list of hooks you can use. How can I make these variables globally in the next app? and, how do you initiate a web3 instance with the given connector from this package?
Thank you for your time :)

**Comment by NoahZinsmeister** [comment]
hey @velea18 , have you looked at the example in this repo? it should give you a good sense of how to get started!
